src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  font-family: $font-family;
+  font-size: $main-font-size;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  &__img {
+    align-self: center;
+    height: 134px;
+    margin-bottom: 40px;
+    width: 160px;
+  }
+
+  &__title {
+    color: $main-accent-color;
+    font-size: 1em;
+    font-weight: 500;
+    line-height: 1.5em;
+    margin: 0;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 140%;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    align-items: end;
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-count {
+    color: $main-accent-color;
+    font-size: 10px;
+    line-height: 140%;
+    margin: 0;
+    text-align: right;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $secondary-color;
+    font-size: 1em;
+    line-height: 150%;
+    margin: 0;
+  }
+
+  &__price-value {
+    color: $main-accent-color;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 112.5%;
+    margin: 0;
+    text-align: right;
+  }
+
+  &__buy-btn {
+    background-color: $blue-accent-color;
+    border: 1px solid $blue-accent-color;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)